inst/pages/beta_diversity.qmd

```diff
@@ -236,6 +236,135 @@ tse <- runMDS(
 plotReducedDim(tse, "Unifrac", colour_by = "Group")
 ```
 
+### Rarefaction to mitigate impacts of uneven sequencing effort
+
+The sequencing depth of a sample refers to the number of gene sequences read 
+during the sequencing process. A variation in sequencing depth across the 
+samples of a study may impact the calculation of alpha and beta diversity 
+metrics [@Schloss2023]. 
+It is common to find significant variation in sequencing depth between samples.
+For instance, the samples of the *TreeSummarizedExperiment* dataset 
+`GlobalPatterns` show up to a 40-fold difference in the number of gene sequences.
+
+```{r rarefaction_1}
+# Calculate the list of sequencing depths across samples
+sequencing_depths <- colSums(assay(tse))
+# Calculate variation between highest and lowest sequencing depth 
+depth_variation <- max(sequencing_depths)/min(sequencing_depths)
+depth_variation
+```
+
+To address uneven sequencing effort, rarefaction aims to normalize sequence 
+counts using subsampling. 
+First, the user chooses the rarefaction depth and a number of iterations N. All the samples with sequence count below the rarefaction depth are removed and gene sequences are randomly drawn from the samples left to get subsamples fitting the rarefaction depth. Then a beta diversity metric is calculated from those subsamples and the process is iterated N times. Finally, beta diversity is estimated with the mean of all beta diversity values calculated on subsampled data.
+
+There has been a long-lasting controversy surrounding the use of rarefaction in
+microbial ecology. The main concern is that rarefaction would omit data. 
```

Comment:
Could you add citations to the following here:

https://journals.asm.org/doi/10.1128/msphere.00355-23
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003531